components/fullscreen-image-quote/manifest.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "http://localhost:3000/schemas/v1.json#",
+  "name": "fullscreen-image-quote",
+  "version": "0.0.6",
+  "namespace": "stanford-components",
+  "description": "This component displays a fullwidth image with a quote and CTA.",
+  "displayName": "Fullscreen Image Quote",
+  "mainFunction": "main",
+  "icon": {
+    "id": "format_quote",
+    "color": {
+      "type": "enum",
+      "value": "red"
+    }
+  },
+  "functions": [
+    {
+      "name": "main",
+      "entry": "./dist/server.cjs",
+      "input": {
+        "type": "object",
+        "properties": {
+          "image": {
+            "title": "Background image (landscape orientation)",
+            "description": "The preferred aspect ratio for the image is 3x2. On the largest screens (> 1500px), the image will be cropped to a 2x1 aspect ratio.",
```

Comment:
Thanks for adding these descriptive descriptions for each field! VERY helpful for editors. 🙏